make-dist

```diff
@@ -61,6 +61,14 @@ download_boost() {
     rm -rf src/boost
 }
 
+build_dashboard_frontend() {
+  pushd src/pybind/mgr/dashboard_v2/frontend
```

Comment:
Does this mean we're building directly in $CEPH_ROOT/src/pybind? Can't we have this building in $CEPH_ROOT/build as well?

Reply:
Yes, the result of the frontend build will reside in `$CEPH_ROOT/src/pybind/mgr/dashboard_v2/frontend/dist`, we cannot change this because:
* it's easy to cope with the way the mgr modules are installed
* the webserver code can find the frontend code without any configuration, both for development and production environments

Reply:
Just for posterity, I'd really like to have all this code being built in `build/` instead. We have a `build/` directory, after all :)